programmers/난이도별/level02.최댓값과_최솟값/areumsheep.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+    var str = s.split(' ');
+    return Math.min.apply(null,str) + ' ' + Math.max.apply(null,str);
```

Comment:
```suggestion
    const strArr = str.split(' ');
    return `${Math.min(...strArr)} ${Math.max(...strArr)}`;
```
이렇게 spread 연산을 사용하면더 간단하게 표현할 수 있답니다!
![image](https://user-images.githubusercontent.com/18749057/92860688-8039c300-f433-11ea-8e48-4b399eaa3e58.png)